### PR TITLE
[Keyboard] Add SIRIND TOMAK

### PR DESCRIPTION
**Pull request by eerraa**
[Keyboard] Add SIRIND TOMAK

## Description
PCB for SIRIND TOMAK
Ergonomics Split Keyboard powered by RP2040.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by waffle87** [comment]
Please do not close and open new PRs when your existing ones would have been fine to continue making changes to. This is now the third PR for this keyboard. (#23241, #22881)

File: keyboards/era/sirind/tomak/info.json [hunk 1]
```diff
@@ -0,0 +1,710 @@
...
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP20"
+    },
+    "layout_aliases": {
+        "LAYOUT_all": "LAYOUT_split_tkl_ansi_split_rshift_bs_rspace",
+        "LAYOUT_tomak_ansi": "LAYOUT_split_tkl_ansi",
+        "LAYOUT_tomak_ansi_split_bs": "LAYOUT_split_tkl_ansi_split_bs",
+        "LAYOUT_tomak_ansi_split_rshift": "LAYOUT_split_tkl_ansi_split_rshift",
+        "LAYOUT_tomak_ansi_split_rshift_bs": "LAYOUT_split_tkl_ansi_split_rshift_bs"
```
Comment (waffle87) [inline keyboards/era/sirind/tomak/info.json:209]: Layout names should not contain the keyboard name in them.

File: keyboards/era/sirind/tomak/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Tomak
+
+![Tomak](https://i.imgur.com/CmVR0G1h.jpeg)
```
Comment (waffle87) [inline keyboards/era/sirind/tomak/readme.md:3]:
```suggestion
![Tomak](https://i.imgur.com/CmVR0G1.jpeg)
```

File: keyboards/era/sirind/tomak/tomak.c [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+        {
+            if (record->event.pressed) {
+                g_tomak_config.indicator_hsv.s = decrement( g_tomak_config.indicator_hsv.s, 8, 0, 255 );
+                write_tomak_config_to_eeprom(&g_tomak_config);
+            }
+            return false;
+        }
+    }
+    return process_record_user(keycode, record);
+};
```
Comment (waffle87) [inline keyboards/era/sirind/tomak/tomak.c:182]: This custom indicator code should be removed.
```suggestion
```

File: keyboards/era/sirind/tomak/tomak.h [hunk 4]
```diff
@@ -0,0 +1,28 @@
+// Copyright 2023 Hyojin Bak (@eerraa)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "quantum.h"
+
+enum tomak_keycodes {
```
Comment (waffle87) [inline keyboards/era/sirind/tomak/tomak.h:8]: As above.

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/era/sirind/tomak/info.json (hunk 1 above)
Comment (eerraa) [reply keyboards/era/sirind/tomak/info.json]: I edited it. :)

File: keyboards/era/sirind/tomak/tomak.h (hunk 4 above)
Comment (eerraa) [reply keyboards/era/sirind/tomak/tomak.h]: I edited it. :)

File: keyboards/era/sirind/tomak/tomak.c (hunk 3 above)
Comment (eerraa) [reply keyboards/era/sirind/tomak/tomak.c]: @waffle87 
Is the most recently modified 'tomak.c' acceptable?

**Review by waffle87 — APPROVED** [review]

**Comment by eerraa** [comment]
Thank you :)

File: keyboards/era/sirind/tomak/keymaps/default/readme.md
Comment (fauxpark) [inline keyboards/era/sirind/tomak/keymaps/default/readme.md:1]: This file can be deleted.

File: keyboards/era/sirind/tomak/keymaps/default_ansi/readme.md
Comment (fauxpark) [inline keyboards/era/sirind/tomak/keymaps/default_ansi/readme.md:1]: This file can be deleted.

File: keyboards/era/sirind/tomak/keymaps/default_ansi_split_bs/readme.md
Comment (fauxpark) [inline keyboards/era/sirind/tomak/keymaps/default_ansi_split_bs/readme.md:1]: This file can be deleted.

File: keyboards/era/sirind/tomak/keymaps/default_ansi_split_rshift/readme.md
Comment (fauxpark) [inline keyboards/era/sirind/tomak/keymaps/default_ansi_split_rshift/readme.md:1]: This file can be deleted.

File: keyboards/era/sirind/tomak/keymaps/default_ansi_split_rshift_bs/readme.md
Comment (fauxpark) [inline keyboards/era/sirind/tomak/keymaps/default_ansi_split_rshift_bs/readme.md:1]: This file can be deleted.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/era/sirind/tomak/keymaps/default_ansi_split_rshift_bs/readme.md
Comment (eerraa) [reply keyboards/era/sirind/tomak/keymaps/default_ansi_split_rshift_bs/readme.md:1]: OK! Keymap readme files have been deleted.

**Comment by eerraa** [comment]
@fauxpark
Could you please check if my correction is correct?
Thank you!

**Review by drashna — APPROVED** [review]

**Pull request merged**